thirdparty/kyaml/runfn/runfn.go

```diff
@@ -110,6 +108,53 @@ func (r RunFns) Execute() error {
 	return r.runFunctions(nodes, output, fltrs)
 }
 
+func (r RunFns) getPipelineConfigFilterFn() (kio.LocalPackageSkipFileFunc, error) {
+	fnConfigPaths := sets.String{}
+	// TODO: eventually walking all sub-packages should be handled by
+	// pkg.FunctionConfigFilePaths
+	err := filepath.Walk(r.absPath, func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return fmt.Errorf("failed to traverse package: %w", err)
+		}
+		if !info.IsDir() {
+			return nil
+		}
```

Comment:
Check for `Kptfile` is missing here.

Reply:
If `Kptfile` is missing, line 129 will fail.